portfolio/src/main/webapp/style.css

```diff
@@ -87,17 +87,44 @@ h4 {
     box-shadow: 0px 10px 30px rgba(57, 56, 61, 0.205);
 }
 
-.primary-button:hover{
+.primary-button:hover {
     background: whitesmoke;
     background-clip: padding-box;
 }
 
-.primary-button::after{
-    position: absolute;
-    top: -2px; left: -2px;
-    bottom: -2px;  right: -2px;
-    background: linear-gradient(90deg, #4458dc 0%, #854fee 100%);
-    content: '';
-    z-index: -1;    
-    border-radius: 4px;
+.primary-button::after {
+  position: absolute;
+  top: -2px; left: -2px;
+  bottom: -2px;  right: -2px;
+  background: linear-gradient(90deg, #4458dc 0%, #854fee 100%);
+  content: '';
+  z-index: -1;    
+  border-radius: 4px;
+}
+/* About Me */
+
+.about {
+  padding: 8rem 5rem;
+}
+
+.about-me {
+  padding-top: 20px;
+}
+
+.about-img {
+  width: 80%;
+}
+
+.about-me::after {
+    content: " ";
```

Comment:
Q: I don't quite understand the purpose of using CSS to add a space after an element either. There may be simpler way to do what this CSS rule is trying to achieve.